docs/client-linux-wireguard.md

```diff
@@ -2,14 +2,9 @@
 
 ## Install WireGuard
 
-To connect to your AlgoVPN using [WireGuard](https://www.wireguard.com) from Ubuntu, first install WireGuard:
+To connect to your AlgoVPN using [WireGuard](https://www.wireguard.com) from Ubuntu, make sure your system is up-to-date then install WireGuard:
```

Comment:
Might be better to just add to the instructions `sudo apt update && sudo apt upgrade`.

Reply:
Done, thanks.